native/cocos/editor-support/spine-creator-support/SkeletonCacheAnimation.h

```diff
@@ -122,6 +125,7 @@ class SkeletonCacheAnimation : public cc::RefCounted, public cc::middleware::IMi
     bool _isAniComplete = true;
     std::string _animationName;
     bool _useTint = true;
+    bool _batch = false;
```

Comment:
considering that it is a bool value, it is better to rename it as ‘canBatch’ or 'isBatch'